explorer/app/components/Header/DrawerContent.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from "@mui/material";
+import React from "react";
+import { LogoProps } from "../Logo/Logo";
+import { NavLinksProps, NavLinks } from "../NavLinks/NavLinks";
+import { SocialLinksProps, SocialLinks } from "../SocialLinks/SocialLinks";
+import {
+  DrawerContainer,
+  SloganDrawerContainer,
+  SocialDrawerContainer,
+} from "./Header.styles";
+
+interface DrawerContentProps {
+  logo: LogoProps;
+  navLinks: NavLinksProps;
+  socialLinks: SocialLinksProps;
+}
+
+export const DrawerContent = ({
+  logo,
+  navLinks,
+  socialLinks,
+}: DrawerContentProps) => {
+  return (
+    <DrawerContainer>
```

Comment:
This component is looking great! I only have a few micro suggestions - to achieve a closer representation of the mock list item spacing we can simplify this component with the following:

- Let's remove the padding around the DrawerContent component, it will then allow for each list item, slogan and socials to take up the full width of the container.
- If we flex or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_grid) the container, we can utilise [gap](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) - this is super handy for creating the necessary spacing between socials or nav items - in this instance we can use it to achieve the 8px gap between the slogan and nav items, and the 8px gap between the nav items and socials.

Reply:
got it! I used to avoid `gap` with `flexbox` for support reasons, but it seems that it isn't necessary anymore, nice!